### ISSUE TITLE
Composite UI Application Block

### ISSUE THREAD

**Issue by mjrousos** [Chavoshi/EnterpriseLibrary.NetCore#36]
@Chavoshi, first of all, thank you for your work making the Enterprise Library components available for NetStandard/NetCore. With .NET Core 3 on the horizon, I've been helping customers migrate WPF and WinForms apps to .NET Core and the Enterprise Library packages are really useful for migrating older desktop apps to run on Core.

Now that WinForms works on .NET Core, a related component that some customers have needed is the [Composite UI Application Block](https://docs.microsoft.com/previous-versions/msp-n-p/ff648747(v=pandp.10)) (CAB). CAB isn't part of the Enterprise Library, but is a related application block, also developed by the Patterns & Practices team. It's similar to Prism, but for WinForms instead of WPF.

My team (Microsoft's DevDiv Customer Engagement Team) is looking into creating an unofficial .NET Core port of CAB to unblock customers who used it previously and would like to run on .NET Core. Like the other Enterprise Library components, it's not something that Microsoft intends to support in any formal capacity, but we'd like to contribute to community efforts to have a .NET Core-compatible CAB package available.

Rather than work in a separate repository, I thought it might make sense to build on the great work you've already done with other Enterprise Library components by adding CAB either here or in github.com/EnterpriseLibrary. Would you be open to us contributing to your repo in this way? If so, my team could do the porting work (which I expect will be small) and create a pull request to add CAB alongside the other Enterprise Library components. While we're at it, we can also enable some other Enterprise Library components for .NET Core that wouldn't have made sense prior to NetCore 3.0 but can be supported now (like Validation.Integration.WPF and Validation.Integration.WinForms).

If it's easier, feel free to reach me on email - MikeRou at Microsoft.com.
